x-pack/plugins/observability_solution/apm/server/routes/transactions/get_span/index.ts

```diff
@@ -47,6 +80,39 @@ export async function getSpan({
             ]),
           },
         },
+        fields: [
```

Comment:
I think we should just use `*` here tbh. this list is unmaintainable and you're bound to miss something.